webhook/resourcesemantics/conversion/reconciler.go

```diff
@@ -65,8 +65,7 @@ func (r *reconciler) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 
 	if !r.IsLeaderFor(types.NamespacedName{Name: key}) {
-		logger.Debugf("Skipping key %q, not the leader.", key)
-		return nil
+		return controller.NewSkipKey(key)
```

Comment:
Keep the log?

Reply:
I intentionally dropped these since we use debug logging in a lot of our CI jobs and it is unnecessarily noisy.